tests/test_fs.py

```diff
@@ -23,16 +23,77 @@
 from v3iofs.path import split_container
 
 
-def test_ls(fs: V3ioFS, new_file):
-    new_file(fs._client, f'{test_dir}/test-file')  # Make sure dir exists
+def test_ls(fs: V3ioFS, tmp_obj):
+
     path = f'/{test_container}/{test_dir}/'
-    out = fs.ls(path)
-    assert len(out) > 0, 'nothing found'
-    assert all(isinstance(p, dict) for p in out), 'not dict'
```

Comment:
Why did you remove this check?

Reply:
The original test included a new_file object created in conftest, and new_file only included a single file one layer deep.  This was removed, and tmp_obj used to be consistent with other tests.  The additional file cases below were added to validate ability to list nested files, and to verify that file details are properly identified.